src/ccmain/control.cpp

```diff
@@ -1364,8 +1367,8 @@ void Tesseract::classify_word_and_language(int pass_n, PAGE_RES_IT *pr_it, WordD
   } else {
     tprintf("no best words!!\n");
   }
-  clock_t ocr_t = clock();
   if (tessedit_timing_debug) {
+    clock_t ocr_t = clock();
     tprintf("%s (ocr took %.2f sec)\n", word_data->word->best_choice->unichar_string().c_str(),
             static_cast<double>(ocr_t - start_t) / CLOCKS_PER_SEC);
```

Comment:
The bad news is that g++ is not clever enough and produces a warning with this PR:
```
src/ccmain/control.cpp:1373:39: warning: 'start_t' may be used uninitialized [-Wmaybe-uninitialized]
```

Reply:
What g++ version?

Reply:
Sorry about that. I will initialize it.

Reply:
But init is exactly wasted cycles again?

Reply:
You need to use `[[maybe_unused]]` attribute here.

Reply:
> What g++ version?

g++ (Debian 12.2.0-14) 12.2.0

Reply:
Interesting to check it on gcc-14

Reply:
_Apple clang version 15.0.0 (clang-1500.3.9.4)_ and _g++-14 (Homebrew GCC 14.1.0_2) 14.1.0_ show the same warning.